addon/components/column-chart/component.js

```diff
@@ -462,17 +498,27 @@ export default BaseChartComponent.extend({
 
         // Choose the tallest bar in the stack (lowest y value) and place the max/min labels above that.
         // Avoids label falling under any bar in the stack.
-        let yValues = [];
+        let yValues = { y: [], height: [] };
         this.get('chart').selectAll('.sub rect.bar').each(function () {
-            yValues.push(parseInt(d3.select(this).attr('y')));
+            yValues.y.push(parseInt(d3.select(this).attr('y')));
+            yValues.height.push(parseInt(d3.select(this).attr('height')));
         });
-        const maxLabelY = Math.min(...yValues);
-
+        const maxLabelY = Math.min(...yValues.y);
+        const maxLabelYHeight = Math.max(...yValues.height);
+
+        // if label collision  detection is turned on, then raise the max and min labels slightly above data values.
+        let raiseLabelBy = 0;
+        if (this.get('labelOptions.labelCollisionResolution') === 'auto') {
+            raiseLabelBy = 10;
+        } else if (this.get('labelOptions.labelCollisionResolution') === 'auto') {
```

Comment:
I think this `if` was unintentionally repeated

Reply:
I left the else if, just in case another mode needs to be implemented along with default, or auto.